src/state/store.ts

```diff
@@ -18,7 +19,7 @@ const persistConfig = {
 
 registerSelectors(portfolioSelectors)
 
-const apiMiddleware = [portfolioApi.middleware, marketApi.middleware, assetApi.middleware]
+const apiMiddleware = [portfolioApi.middleware, marketApi.middleware, assetApi.middleware, logging]
```

Comment:
My pedantic self as well as my love for the OCD fam would call that one `loggingMiddleware` but that's just nit-picking